### ISSUE TITLE
Vervang de CLI parameter createyaml door een correcte naam

### ISSUE THREAD

**Issue by ArjanLoeffen** [Imvertor/Imvertor-Maven#232]
De CLI parameter

`createyaml = yes`

heeft niet de goede naam. "Yaml" is niet gebonden aan de API generatie. Overweeg deze te vervangen door een inhoduelijke naam zoals 

`createoas = 3.0'

of 

`createoas - none`

of iets dergelijks (cf. `createoffice - html' versus 'createoffice = none').

**Comment by melsk-r** [comment]
Mee eens om 'createyaml' te vervangen door 'createoas'. 
Gebruik van 'createoas = 3.0' impliceert dat we in staat willen zijn om een keuze te maken tussen OAS versies. Dat lijkt me een redelijke aanname aangezien niet alle organisaties tegelijkertijd over zullen willen gaan naar een nieuwe OAS versie. Iedere organisatie zal daarvoor zijn eigen roadmap hanteren.

In de stylesheets in 'src\main\resources\xsl\YamlCompiler' zal ik vooralsnog geen gebruik maken van deze CLI parameter pas als ik specifieke structuren voor een nieuwe versie moet gaan genereren zal ik checken of die nieuwe versie ook gevraagd wordt m.b.v. de CLI parameter.

I.v.m. dit issue kun je jezelf de vraag stellen of we de folder 'src\main\resources\xsl\YamlCompiler' niet ook moeten hernoemen naar 'src\main\resources\xsl\OasCompiler' en of we in de namen van enkele stylesheets niet ook 'Yaml' moeten vervangen door 'Oas'.

**Comment by ArjanLoeffen** [comment]
Ja, dat lijkt me wel. Dus laten we daar dan een moment voor kiezen. Doen we het allemaal in één klap,.

**Comment by ArjanLoeffen** [comment]
@melsk-r Zullen we dit maar eens gaan doen? En dan moeten we "oas" nader specificeren als de VNG versie van OAS, omdat we misschien een tweede methode gaan opnemen in Imvertor.
